tomls/omnibus-arbitrum-mainnet/spot/WSOL.toml

```diff
@@ -0,0 +1,84 @@
+[var.WSOL_spot_settings]
+synth_WSOL_max_collateral_amount = "<%= parseEther(String(7_500)) %>"
+synth_WSOL_skew_scale = "<%= parseEther(String(1_406_250)) %>"
+
+[invoke.SpotMarketProxy_createSynth_WSOL]
+target = ["spotFactory.SpotMarketProxy"]
+fromCall.func = "owner"
+func = "createSynth"
+args = ["Synthetic Wormhole SOL", "swSOL", "<%= settings.owner %>"]
```

Comment:
I think `sWSOL` would be more consistent naming for this one

Reply:
isn't it stBTC ?

Reply:
If you look at the actual token for `tBTC`, this is how it is named at it's `symbol()` on the ERC20 contract. For `WSOL` the symbol is actually `SOL`, similar to how we have a `WETH` token but our synth is named `ETH`.

I think it's best to be consistent across all references. There are references for `swSOL`, `wSOL` and `WSOL` throughout various file names and variable names.

Reply:
The symbol is actually `SOL` so it all changed to `sSOL` now. There is no `w`

Reply:
![image](https://github.com/user-attachments/assets/15a5f33f-6014-487a-b92c-9864817b094d)